composer.json

```diff
@@ -11,18 +11,22 @@
   ],
   "minimum-stability": "stable",
   "require": {
-    "php": ">=7.0",
+    "php": "^7.0 || ^8.1",
     "simplesamlphp/simplesamlphp": "~1.18.6 || ~1.19.0"
   },
   "require-dev": {
-    "phpunit/phpunit": "^5.6.1",
+    "phpunit/phpunit": ">=7.0",
```

Comment:
Given that PHPUnit is 9.x already, wouldn't >=8.0 be better?

Reply:
I think we want to continue supporting PHP 7.4 as long as it's still an officially supported version of PHP